File: C-Python_Solving_50_Problems/13-Max_Number_Of_Three.py
# Python program to find the maximum of three numbers
# take inputs
num1 = float(input('Enter first number: '))
num2 = float(input('Enter second number: '))
num3 = float(input('Enter third number: '))

# find maximum
max_num = max(num1, num2, num3)

# display result
print('The maximum of {0}, {1}, and {2} is {3}'.format(num1, num2, num3, max_num))

File: C-Python_Solving_50_Problems/13-Max_Number_Of_Three.c
#include <stdio.h>

int main() {
    float num1, num2, num3, max_num;

    // take inputs
    printf("Enter first number: ");
    scanf("%f", &num1);
    printf("Enter second number: ");
    scanf("%f", &num2);
    printf("Enter third number: ");
    scanf("%f", &num3);

    // find maximum
    if (num1 > num2) {
        if (num1 > num3) {
            max_num = num1;
        } else {
            max_num = num3;
        }
    } else {
        if (num2 > num3) {
            max_num = num2;
        } else {
            max_num = num3;
        }
    }

    // display result
    printf("The maximum of %.2f, %.2f, and %.2f is %.2f", num1, num2, num3, max_num);

    return 0;
}
